### DISCUSSION TITLE
是否可以设置mongodb的查询查secondary？

### DISCUSSION THREAD

**Discussion by pirate4444** [hhyo/Archery#2817] [Q&A]
目前我们使用的版本是1.8.5，想限制查询mongodb时，是查secondary，而不是查primary。但是找不到在哪里进行相关的设置（readPreference: 'secondary'）。是否支持查secondary？

**Comment by feiazifeiazi** [comment]
曲线救国：
配置时用单个IP。 这个IP对应的就是secondary，  如果mongo切换了，需要人工改一下。

我的SQL上线工单，mongo就是配单个IP地址，mongog切换了，得改地址才行。
